packages/canner/test/hocs/validation.test.js

```diff
@@ -88,7 +88,7 @@ describe('withValidation', () => {
       {...props}
       onDeploy={jest.fn().mockImplementation((_, fn) => (fn(result)))}
```

Comment:
Actually we don't have to mock the `onDeploy` method to execute the validation, just get the wrapper instance(see the next comment).
```suggestion
      onDeploy={jest.fn().mockImplementation((_, fn) => (fn(result)))}
```